src/main.js

```diff
@@ -5,16 +5,21 @@ import createMovieWrapper from './view/movie-wrapper';
 import createSortCardTempale from './view/sort';
 import createShowMoreButton from './view/show-more-button';
 import createFilmDetailsTempale from './view/film-details';
-import render from './utils';
+import { render } from './utils';
+import { generateCardFilm } from './mock/card-film';
 
-const CARD_COUNT = 5;
+const FILM_COUNT = 20;
+const TASK_COUNT_PER_STEP = 5;
 const CARD_COUNT_EXTRA = 2;
 
+// eslint-disable-next-line max-len
+const filmCards = new Array(FILM_COUNT).fill().map((element, idElement) => generateCardFilm(idElement));
```

Comment:
Можно избавиться от комментария `// eslint-disable-next-line max-len` если по другому написать код? 
```suggestion
const filmCards = new Array(FILM_COUNT)
    .fill()
    .map((element, idElement) => generateCardFilm(idElement));
```

Reply:
fixed